deploy/readme.md

```diff
@@ -0,0 +1,32 @@
+# Deploy 
+
+## Deploy Components
+
+Reusable components for deployment and/or data migration. 
+
+### Migrate 
+  Base instructions used for data migration per environment using flyway and Postgres.  
+#### Example
+```yaml 
+  migrate-int:
+  extends: .migrate
+  stage: migrate-int
+```
+
+### Deploy
+
+Base instructions used for deployment. 
```

Comment:
I feel like this line isn't super helpful.

Maybe something about extensible abstract tasks with a link to the gitlab ci documentation for those that aren't familiar with how gitlab ci extension works?